src/utils/deploy/templates.ts

```diff
@@ -187,9 +189,10 @@ const deployIndividualTemplates = async (
     } else {
       logline(chalk.greenBright(`Template ${template.friendlyId} does not exist in the tenant - will create`));
 
-      template.status = 'PUBLISHED';
+      // template.status = 'PUBLISHED';
```

Comment:
This one will at some point require further discussion.  I agree probably not a good idea to enforce this, but some customers may want to because really in my mind I'm not sure this should even be a state here, when source control is involved, because it feels counter intuitive, because if you deploy it and it's not ready for a marketer to use, the marketer may go in and change it, only to find in the next deploy that it's now overwritten.  Also then when it's ready it's got to go through a deploy and source control change, which doesn't feel right either.  Thinking from the XP world, you don't really have Templates that are draft, content maybe would be draft but not the templates themselves.  It implies they aren't ready, and in my mind if they aren't ready, should you be deploying these to a tenant?  But either way, maybe just a setting is needed here.  I was thinking maybe a setting.json that sits in the root of the artifacts could have some basic config like this.  For now, since this is experimental software, lets move ahead with this, and I'll create an issue that we can discuss options, including the settings item to control this.

Also a little side not, you may notice weird logic in there about doing an update to change draft state to Draft, before you can make changes to it, and then setting it back to published once you run the changes update.  Will talk to product about that scenario.

Reply:
That makes sense and I agree, we need to discuss further.

This removal of the overwrite is only on create though and I'm thinking that if it's a new item, just go ahead and get the status from the file.